server/queries/createSurvey.js

```diff
@@ -8,13 +8,14 @@ const createSurvey = async (surveyObj, newQuestionsArray) => {
   const surveysCollection = db.collection('Surveys');
   const newSurveyObj = { ...surveyObj, questions: newQuestionsArray };
   try {
-    const { acknowledged, surveyId } = await surveysCollection.insertOne(
-      newSurveyObj,
-    );
-    if (acknowledged === false) {
+    const queryResult = await surveysCollection.insertOne(newSurveyObj);
+    console.log('Survey added to DB!');
+
+    if (queryResult.result.ok !== 1) {
```

Comment:
can you save this condition into a more declarative var?

Reply:
will fix this when I merge my stuff in